web/src/containers/ActivityDetailCostAllocate.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Editor } from 'react-draft-wysiwyg';
+import { connect } from 'react-redux';
+
+import { updateActivity as updateActivityAction } from '../actions/activities';
+import Collapsible from '../components/Collapsible';
+import { Input } from '../components/Inputs2';
+import { EDITOR_CONFIG, htmlToEditor, editorToHtml } from '../util/editor';
+
+class ActivityDetailCostAllocate extends Component {
+  constructor(props) {
+    super(props);
+
+    const { costAllocateDesc, otherFundingDesc } = props.activity;
+
+    this.state = {
+      costAllocateDesc: htmlToEditor(costAllocateDesc),
+      otherFundingDesc: htmlToEditor(otherFundingDesc)
+    };
+  }
+
+  onEditorChange = name => editorState => {
```

Comment:
we can/should create a HOC when there's an editor component so that these methods/component state can be abstracted away

Reply:
`<RichTextarea>`  😛